src/index.js

```diff
@@ -0,0 +1,245 @@
+  getProjects();
+  selectOption();
+  displayCurrentProjects(currentProject);
+};
+
+const createProjectName = (project) => {
+  const newProject = projectObject(project);
+  allProjects.push(newProject);
+  saveProjects();
+  domObjects.errorMsgsAlert('the project has been created successfully');
+  selectProject.innerHTML = '';
+  selectOption();
+  domObjects.hideProjectForm();
+  currentProject = selectProject.value;
+};
+
+const createTodoObject = (title, description, dueDate, priority) => {
+  if (currentProject == '') {
+    createProjectName('Default Project');
+  }
+
+  const newTodo = todoObject(title, description, dueDate, priority);
+  allProjects.forEach((project) => {
+    if (project.projectName === currentProject) {
+      project.todoList.push(newTodo);
+      saveProjects();
+      displayCurrentProjects();
+      domObjects.hideTodoForm();
+    }
+  });
+};
+
+const validateProjectInput = (event) => {
+  event.preventDefault();
+  if (projectName.value === '') {
+    domObjects.errorMsgsAlert('Project Name can not be empty');
+  } else {
+    createProjectName(projectName.value);
+  }
+};
+
+const validateTodoInput = (event) => {
+  event.preventDefault();
+  if (todoTitle.value === '') {
+    domObjects.errorMsgsAlert('Title can not be empty');
+  } else {
+    createTodoObject(todoTitle.value, todoDescription.value, todoDueDate.value, todoPriority.value);
+  }
+};
+
+const deleteTodo = (target) => {
+  allProjects.forEach((proj) => {
+    if (proj.projectName === currentProject) {
+      proj.todoList.splice(target.value, 1);
+      saveProjects();
+      displayCurrentProjects(currentProject);
+    }
+  });
+};
+
+const showEditTodoForm = (target) => {
+  domObjects.showTodoForm();
+  domObjects.hideSubmitTodo();
+  allProjects.forEach((proj) => {
+    if (proj.projectName === currentProject) {
+      const temp = proj.todoList[target.value];
+      todoTitle.value = temp.title;
+      todoDescription.value = temp.description;
+      todoDueDate.value = temp.dueDate;
+      todoPriority.value = temp.priority;
+      currentTodo = target.value;
+    }
+  });
+};
+
+const editTodoObject = () => {
+  domObjects.showSubmitTodo();
+  domObjects.hideTodoForm();
+  allProjects.forEach((proj) => {
+    if (proj.projectName === currentProject) {
+      const editedTodo = todoObject(todoTitle.value, todoDescription.value, todoDueDate.value, todoPriority.value); // eslint-disable-line 
+      proj.todoList[currentTodo] = editedTodo;// eslint-disable-line 
+      const objIndex = allProjects.findIndex(((obj) => obj.projectName == currentProject));
+      allProjects[objIndex] = proj;
+      saveProjects();
+      displayCurrentProjects(currentProject);
+    }
+  });
+};
+
+const btnListner = (targetBtn) => {
+  if (targetBtn.classList == 'editBtn') {
+    showEditTodoForm(targetBtn);
+  } else if (targetBtn.classList == 'deleteBtn') {
+    deleteTodo(targetBtn);
+  }
+};
+
+initialLoad();
+// console.log(initialLoad());
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history.